### ISSUE TITLE
Overidding sails app for production environment

### ISSUE THREAD

**Issue by Aizenko** [balderdashy/sails-docs#293]
Hi,

I'm testing sails. My usual dev setup is based on subversion which I use to deploy my projects on production server.
I have a config file for const and dabase connection infos which include (if it exists) an local file which is used to overide some of the variables according to the deployment env. Such as new credential for the database. This file is not under svn so when the project is upgraded this settings are not overrided.
How can I have the same behavior with sails ?

**Comment by sailsbot** [comment]
Thanks for posting, @Aizenko.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
